lib/perf_hooks.js

```diff
@@ -640,6 +641,7 @@ function sortedInsert(list, entry) {
 }
 
 class ELDHistogram extends Histogram {
+  constructor(i) { super(i); } // eslint-disable-line no-useless-constructor
```

Comment:
I know this has been discussed elsewhere but I can't remember the reason we're doing this.

Reply:
Yeah, that's a bug in the ECMAScript spec, it should be fixed by https://github.com/tc39/ecma262/pull/2216.

Relevant spec section: https://tc39.es/ecma262/#sec-runtime-semantics-classdefinitionevaluation

> If constructor is empty, then
> >If ClassHeritageopt is present, then
> > > Let constructorText be the source text
> > > > constructor(...args) { super(...args); }